Solutions/GoogleCloudPlatformIAM/Hunting Queries/GCPIAMChangedRoles.yaml

```diff
@@ -0,0 +1,26 @@
+id: 0ab42aac-2c86-443c-80fa-ef8cdd0d997e
+name: GCP IAM - Changed roles
+description: |
+  'Query searches for service accounts disabled for the last 24 hours.'

```

Comment:
The name and the description seems to be confusing.  It seems that the query is showing all the changed/updated roles and not just the disabling of Service Accounts. Wondering, if we need to re-word the description.